### ISSUE TITLE
Accessibility checks fail

### ISSUE THREAD

**Issue by schwartzadev** [alexwolfe/Buttons#82]
Feel free to confirm [here](https://webaim.org/resources/contrastchecker/).

You can also check on the Google Chrome audits, under accessibility.
